README.md

```diff
@@ -237,6 +237,38 @@ try {
 }
 ```
 
+### Toggle Climatisation
+
+To toggle climatisation mode to on or off. This call will return a `RemoteCommandAccepted` struct when the request has been accepted. 
+
+The `enable` paramater is optional and defaults to true. 
+
+```swift
+try {
+  let result = porscheConnect.toggleClimatisation(vehicle: vehicle, enable: false)
+  if let remoteCommandAccepted = result.remoteCommandAccepted {
+    // Do something with the remote command
+  }
+} catch {
+  // Handle the error
+}
+```
+
+As Toggle Climatisation is a remote command that can take time to reach and be executed by the car, you can check the status of the command. You pass in both the vehicle and the response from the `toggleClimatisation()` call above. 
```

Comment:
It's referred to as "Toggle Climatisation" here but "Toggle Direct Climatisation" elsewhere. Does the "direct" portion have a specific meaning in the context of this API? If not, consider dropping from everywhere but the network route where it's needed.